libphx/src/lua.rs

```diff
@@ -84,12 +84,12 @@ static mut activeInstance: *mut Lua = std::ptr::null_mut();
 static mut cSignal: Signal = 0;
 
 unsafe extern "C" fn Lua_BacktraceHook(this: *mut Lua, _: *mut lua_Debug) {
+    let msg = CString::new(format!("Received Signal: {}", signal_to_string(cSignal))).unwrap();
+
     lua_sethook(this, None, 0, 0);
     luaL_where(this, 0);
-    lua_pushstring(
-        this,
-        StrAdd(c_str!("Received Signal: "), Signal_ToString(cSignal)),
-    );
+    // TODO: check this doesn't cause memory double free bug, do 'forget' if it is
```

Comment:
I'm pretty certain `lua_pushstring` does not free any memory, because it can be used with static constants like `lua_pushstring(this, "Hello World")`. I think it's fine to remove the comment.

Reply:
Removed